README.md

```diff
@@ -2,165 +2,39 @@
 
 ## Overview
 
-The Google Play Instant Plugin for Unity Beta simplifies conversion of a Unity-based Android app into an instant app that can be deployed through [Google Play Instant](https://developer.android.com/topic/google-play-instant/).
+This GitHub project is now archived, and the code has been migrated to the
+[Play Unity Plugins](//github.com/google/play-unity-plugins) project.
 
-The plugin’s Unity Editor (IDE) features include:
+Refer to the
+[developer documentation](//developer.android.com/topic/google-play-instant/getting-started/game-unity-plugin)
+for the latest information about this plugin.
 
- * The option to switch between Installed and Instant build modes.
- * A centralized view of Android Player Settings that should be changed to support Google Play Instant.
- * An action to build the instant app for publishing on Google Play Console.
- * An action to build and run the instant app on an adb connected Android device.
+## Migration details
 
-The plugin’s Unity Engine (runtime) features include:
+Some of the files in this archived GitHub project have the same Guids as files
+in the new GitHub project. This allows projects that are using Play assets, such
+as the `LoadingScreen` MonoBehaviour, to still work after migrating.
 
- * A method for displaying a Play Store dialog to install the full version of the instant app.
+The new **Play Unity Plugins** project uses the namespace `Google.Play.Instant`
+instead of `GooglePlayInstant`, so any `using GooglePlayInstant;` statements
+will have to be changed.
 
-## Installing the Plugin
+## Migration steps
 
-### Prerequisites
+1.  Familiarize yourself with the
+    [download and import](//developer.android.com/topic/google-play-instant/getting-started/game-unity-plugin#import-plugin)
+    process for the new plugin, and either download the new `.unitypackage` file
+    or set up the **Game Package Registry for Unity**.
 
- * Unity 5.6, 2017.4, or 2018.4.
-   * Note: other versions may work, but are not tested as regularly as these.
- * A device running Android 5.0 (Lollipop) or newer.
+1.  Delete the existing `Assets/GooglePlayInstant` directory (if you previously
```

Comment:
Do these numbers need to be updated or does markdown change them automatically?

Reply:
Markdown handles this, for example see: https://github.com/google/play-unity-plugins/blob/master/README.md#for-unity-56-to-20183